Tutorials/Creating-Basic-Site/Creating-Your-First-Template-and-Content-Node/index.md

```diff
@@ -4,60 +4,68 @@ versionFrom: 8.0.0
 
 # Creating (Editing) Your First Template
 
-Next click the expand node icon (it's the small triangle) in front of the **_Settings >_** **_Templates folder_** - you should then see a child node titled "_Homepage_"
+Next click the expand node icon (it's the small triangle) in front of the [Settings] > [Templates folder] - you should then see a child **_node_** titled "Homepage"
 
-Umbraco created this automatically when we created the **_Document Type_** (remember that option that said "..without template", we chose the one with and this is what it did).
+Umbraco created this automatically when we created the **_document type_** (remember that option that said "..without template", we chose the one with and this is what it did).
```

Comment:
A couple of times you've changed "Document Type" to "document type" .. as it's a concept within Umbraco, we prefer writing it with capital letters 😄

```suggestion
Umbraco created this automatically when we created the **_Document Type_** (remember that option that said "..without template", we chose the one with and this is what it did).
```

Reply:
All concepts should be capatilized then?
Document Type
Content Node
Template
... ?

Reply:
Ah, good question!
I'd say both Document Type and Template should be capitalized.
Let's leave content node as is for now, I think!